tests/test-cases.js

```diff
@@ -1475,7 +1580,7 @@ describe('Parse Tests', function() {
 			if (test.expected.meta) {
 				assert.deepEqual(actual.meta, test.expected.meta);
 			}
-			assert.deepEqual(JSON.stringify(actual.errors), JSON.stringify(test.expected.errors));
+			assert.deepEqual(JSON.parse(JSON.stringify(actual.errors)), test.expected.errors);
```

Comment:
Why changing this?

Reply:
as covered in the explanation, JSON does not guarantee ordering or the properties within an object so doing a string compare of two JSON strings is not a good practice.  The goal of the JSON hack is to eliminate the `undefined` properties which deepEqual strictly would fail this on.  I have addressed in a separate branch making PapaParse not produce the undefined value so that this wouldn't be needed in that branch.
This modification keeps the value assertion `test.expected.errors` left untampered while using the JSON workaround for its intended purpose and avoiding the ordering problem.